Rain-Drops/style.css

```diff
@@ -0,0 +1,619 @@
+  font: bold 20px Arial;
+  color: rgb(42, 65, 107);
+}
+.management-game__enter-keys {
+  margin-top: 30%;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  background: linear-gradient(to bottom, #eeeeee77 50%, #b2d2f071 90%);
+  border-radius: 5px;
+}
+
+.display {
+  width: calc(100% - 10px);
+  height: 70px;
+  text-align: center;
+  background: linear-gradient(to bottom, #ffffff, rgb(67, 121, 172));
+  margin: 3px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font: bold 30px Arial;
+  color: rgb(28, 28, 102);
+  border-radius: 5px;
+  overflow: hidden;
+}
+.keys {
+  flex-grow: 1;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 5px;
+  margin: 3px;
+}
+.key {
+  background: linear-gradient(to bottom, #dedeff 20%, #8cbfee 90%);
+  outline: none;
+  border: 1px solid #3450c9;
+  color: rgb(28, 28, 102);
+  font: bold 16px Arial;
+  appearance: button;
+  border-radius: 5px;
+}
+
+.key:focus {
+  border: 1px solid #ffffff;
+}
+.key:hover {
+  border: 1px solid #3450c9;
+}
+.key:active {
+  border: 1px solid #ffffff;
+  background: linear-gradient(to bottom, #dbdbeb 20%, #4096e7 90%);
+}
+.clear {
+  grid-row: span 2 / auto;
+  font: bold 12px Arial;
+}
+.enter {
+  grid-row: span 2 / auto;
+  font: bold 12px Arial;
+}
+.zero {
+  grid-column: span 2 / auto;
+}
+.delete {
+  font: bold 12px Arial;
+}
+.block-result-statistics {
+  width: 100vw;
+  height: 100vh;
+  position: absolute;
+  top: 0px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.616);
+transition: opacity 0.5s linear;
+opacity: 0;
+visibility: hidden;
+}
+.result-current-game {
+  position: relative;
+  background: linear-gradient(to bottom, #506caa 50%, #000436 90%);
+  box-shadow: 0 0 50px rgb(14, 16, 117);
+  border-radius: 50%;
+  width: 280px;
+  height: 280px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+.result-game__information {
+  text-align: center;
+}
+.result-game__information > h2 {
+  margin-bottom: 10px;
+  color: rgb(236, 241, 253);
+}
+.result-game__information > h2:last-child {
+  margin-top: 20px;
+  margin-bottom: 0px;
+}
+.information {
+  text-align: end;
+  font: bold 14px Arial;
+}
+.btn{
+  font: bold 16px Arial;
+  background: linear-gradient(to bottom, #506caa 50%, #000436 90%);
+  color: #dbdbeb;
+  border-radius: 30px;
+  border: none;
+  width: 100px;
+  height: 30px;
+}
+.game-statistics {
+  position: absolute;
+  top: calc(50%-140px);
+  width: 280px;
+  height: 280px;
+  background: linear-gradient(to bottom, #506caa 50%, #000436 90%);
+  box-shadow: 0 0 50px rgb(14, 16, 117);
+  border-radius: 50%;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transition: opacity 0.5s linear;
+  opacity: 0;
+  visibility: hidden;
+}
+.active {
+  opacity: 1;
+  visibility: visible;
+}
+.game-statistics > h2 {
```

Comment:
Will be better to create a class name for headers.
For example, I see that you duplicate the color for the same header.